cmd/dep/check.go

```diff
@@ -58,18 +59,18 @@ func (cmd *checkCommand) Register(fs *flag.FlagSet) {
 	fs.BoolVar(&cmd.quiet, "q", false, "Suppress non-error output")
 }
 
-func (cmd *checkCommand) Run(ctx *dep.Ctx, args []string) error {
-	logger := ctx.Out
+func (cmd *checkCommand) Run(ctx context.Context, depCtx *dep.Ctx, args []string) error {
```

Comment:
What's the difference between `ctx` and `depCtx` here?

Reply:
or I guess, they have different API's, but why is the `depCtx` insufficient here? it doesn't seem like we are using `ctx` at all.

Reply:
The `checkCommand` does not use the `context.Context`, but it must implement the `command` interface.